api-gateway-sqs-service/main.tf

```diff
@@ -88,4 +41,11 @@ resource "aws_cognito_resource_server" "resource_server" {
   }
 
   user_pool_id = var.user_pool_id
+}
+
+module "api-gateway-sqs-roles-duty-change" {
+  source       = "../api-gateway-sqs-roles"
```

Comment:
I should refer to local source, not github ref?

Reply:
Yes, for sources in the same repo you should use the local source